international-journal-of-geriatric-psychiatry.csl

```diff
@@ -45,6 +45,7 @@
   <macro name="author-short">
     <names variable="author">
       <name form="short" and="text" delimiter=", " delimiter-precedes-last="never" initialize-with=". "/>
+      <et-al/>
```

Comment:
why are you adding this? This shouldn't change anything.

Reply:
Not sure where that is coming from. thanks for spotting it.